exercises/isbn-verifier/lib/example.dart

```diff
@@ -0,0 +1,19 @@
+class IsbnVerifier {
```

Comment:
I would suggest removing the class and just allowing the method to be accessible.